World/World/FavoritesViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  FavoritesViewController2.swift
+//  World
+//
+//  Created by Timmy Li on 2/12/20.
+//  Copyright © 2020 Timmy Li. All rights reserved.
+//
+
+import UIKit
+
+class FavoritesViewController: UIViewController, UITableViewDelegate, UITableViewDataSource{
+
+    @IBAction func closeButtonPress(_ sender: Any) {
+        dismiss(animated: true, completion: nil)
+    }
+    @IBOutlet var closeButtonView: UIButton!
+    @IBOutlet var favoritesTableView: UITableView!
+    
+    weak var delegate: PlacesFavoritesDelegate?
+    var favoritePlaces = [String:Int]()
+    
+    lazy var favoritesNameList = {
+        Array(self.favoritePlaces.keys)
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        favoritePlaces = DataManager.sharedInstance.favoritePlaces
+        favoritesTableView.dataSource = self
+        favoritesTableView.delegate = self
+        favoritesTableView.register(UITableViewCell.self, forCellReuseIdentifier: "favoriteRow")
+    }
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
```

Comment:
Use pragma marks to help organize code.

Ex: `MARK: - UITableViewDelegate` can be a section housing all of your code related to implementing `UITableViewDelegate` methods.